2017-11-28 19:39:26 [scrapy.utils.log] INFO: Scrapy 1.4.0 started (bot: Day1126)
2017-11-28 19:39:26 [scrapy.utils.log] INFO: Overridden settings: {'BOT_NAME': 'Day1126', 'LOG_FILE': 'logs\\111\\myHangban\\c7208dbed43011e7b8106807158b033f.log', 'NEWSPIDER_MODULE': 'Day1126.spiders', 'SPIDER_MODULES': ['Day1126.spiders']}
2017-11-28 19:39:26 [scrapy.middleware] INFO: Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.logstats.LogStats']
2017-11-28 19:39:26 [twisted] CRITICAL: Unhandled error in Deferred:
2017-11-28 19:39:26 [twisted] CRITICAL: 
Traceback (most recent call last):
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\site-packages\twisted\internet\defer.py", line 1386, in _inlineCallbacks
    result = g.send(result)
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\site-packages\scrapy\crawler.py", line 77, in crawl
    self.engine = self._create_engine()
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\site-packages\scrapy\crawler.py", line 102, in _create_engine
    return ExecutionEngine(self, lambda _: self.stop())
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\site-packages\scrapy\core\engine.py", line 69, in __init__
    self.downloader = downloader_cls(crawler)
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\site-packages\scrapy\core\downloader\__init__.py", line 88, in __init__
    self.middleware = DownloaderMiddlewareManager.from_crawler(crawler)
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\site-packages\scrapy\middleware.py", line 58, in from_crawler
    return cls.from_settings(crawler.settings, crawler)
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\site-packages\scrapy\middleware.py", line 34, in from_settings
    mwcls = load_object(clspath)
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\site-packages\scrapy\utils\misc.py", line 44, in load_object
    mod = import_module(module)
  File "c:\users\张霄港\appdata\local\programs\python\python36-32\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 936, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 948, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'Day1126.mymiddlewares'
